### PR TITLE
Reduce http requests

### PR DESCRIPTION
**Pull request by XhmikosR**
I also made them responsive.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Before: 336 requests 1,7 MB


---

After: 107 requests 1,3 MB

File: index.html [hunk 1]
```diff
@@ -48,19 +48,24 @@ <h1 id="join-the-revolution">Join the Revolution</h1>
...
 
   <div class="container">
-    <div id="orgs">
-      <div class="the-orgs">
+    <div class="row">
+      <div class="span12">
+    {% assign count = "0" %}
     {% for type_hash in site.organizations %}
       {% for org in type_hash[1] %}
+        {% capture diff %}{{ count | minus:44 }}{% endcapture %}
```
Comment (benbalter) [inline index.html:56]: What exactly are you trying to do here?

**Comment by benbalter** [comment]
I believe part of the initial design vision was to have the orgs overflowing out of the box to indicate that there were many, many more. Can still limit, but should most likely be expanded by one additional row in each direction to maintain that effect.

File: index.html (hunk 1 above)
Comment (XhmikosR) [inline index.html:56]: 44 is the number of organizations shown; 4x11

**Comment by XhmikosR** [comment]
Well, currently that row isn't responsive at all. Personally, I prefer this way, but feel free to improve it after merging this.

**Pull request closed without merging**